.github/workflows/python_dbapi_wheels.yml

```diff
@@ -0,0 +1,104 @@
+name: "Build Python DB API Driver Wheels"
```

Comment:
Is this workflow a part of the release procedure? Should we update RELEASE.md accordingly?

Reply:
It's going to be, yes. Will update.

Reply:
I've decided to implement publishing in another suite and update the release documentation in this ticket:
https://issues.apache.org/jira/browse/IGNITE-24899